doc/api/fs.markdown

```diff
@@ -122,13 +122,13 @@ Stop watching for changes on the given `fs.FSWatcher`.
 
 ### Event: 'open'
 
-* `fd` {Number} Integer file descriptor used by the ReadStream.
+* `fd` {Integer} Integer file descriptor used by the ReadStream.
 
 Emitted when the ReadStream's file is opened.
 
 ### readStream.path
 
-The path to the file the stream is reading from.
+The path to the file the stream is reading from. Can be a string or a `Buffer`.
```

Comment:
It's based on what you pass in to createReadStream/createWriteStream. If you pass a string, it'll be a string. If you pass a buffer it'll be a buffer.